epersons.md

```diff
@@ -12,6 +12,7 @@
 ### Replace
 The replace operation allows to replace *existent* information with new one. Attempt to use the replace operation to set not yet initialized information must return an error. See [general errors on PATCH requests](patch.md)
 
+#### These operations can be performed by administrators.
 
 To replace the certificate required value, `curl -X PATCH http://${dspace.url}/api/epersons/eperson/<:id-eperson> -H "Content-Type: application/json" -d '[{ "op": "replace", "path": "/certificate", "value": "true|false"]'`.  The operation also requires an Authorization header.
```

Comment:
Shouldn't the examples be updated to be using the endpoint?
`/api/eperson/epersons`